tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -139,27 +139,23 @@ vic-machine create grants ops-user perms
 
     Run privilege-dependent docker operations
 
-    Cleanup VIC Appliance On Test Server
+    [Teardown]  Cleanup VIC Appliance On Test Server
```

Comment:
one thing I'm not very sure about: does this teardown overwrite the global teardown `Run Keyword If Test Failed  Gather vSphere Logs`?

Reply:
It would override, but since we don't really need the vsphere logs from these tests, we can get rid of the global teardown.